### ISSUE TITLE
Search for Product in Coupons does not search by product variation title

### ISSUE THREAD

**Issue by johndcoy** [woocommerce/woocommerce#12483]
## EXPLANATION

When searching for a product to include or exclude from a coupon, the search field does not index the variation title only the parent product.

Here you will see _tart_ as an option shown when searching for Pumpkin but not when using the word tart

![Screenshot](https://cloudup.com/cfaCGKWZTBj+)

## STEPS TO REPRODUCE

1. Create a variable product with a unique name from the parent product.
2. Search for that specific variation in the coupon restrictions tab.

## EXPECTATION

To be able to search by products variation title from coupon restrictions.

**Comment by mikejolley** [comment]
Variations do not have meaningful titles. What you're seeing is a formatted list of attributes. Due to the nature of these, it is not indexable.

2.7.x saves attribute as part of the title. It will resolve this but only after the variations are re-saved so a meaningful title is saved. This is a wontfix item until then.
